package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "5.4.5",
+  "version": "5.4.6",
```

Comment:
Eve, do you have an upcoming release for Namerequest UI? Can you please add this ticket to that release? Thanks!

Reply:
could you update the version to 5.4.7? I merged my code with version = "5.4.6" just now,

Reply:
Done!